bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/internal/ImageList.java

```diff
@@ -359,10 +359,14 @@ private void addPlaceholderImageToImageList(long imageListHandle, int bitmapWidt
 	OS.ReleaseDC(0, hDC);
 }
 
+/**
+ *
+ * @return size of the ImageList in points
```

Comment:
It's not the size of the list but the size of images in that list, isn't it?
You could also use `{@return ...}` to show the statement as both the documentation and the return value.